src/languages/php.js

```diff
@@ -102,11 +103,50 @@ export default function(hljs) {
     // <https://www.php.net/manual/en/reserved.classes.php>
     'Directory __PHP_Incomplete_Class parent php_user_filter self static stdClass'
   };
+  const ATTRIBUTES = {
+    className: 'meta',
+    begin: '^\\s*#\\[',
+    end: '\\]',
+    contains: [
+      {
+        begin: /\[/,
+        end: /\]/,
+        contains: [
+          'self',
+          STRING,
+          hljs.C_BLOCK_COMMENT_MODE,
+          STRING,
+          NUMBER,
+          {
+            className: 'literal',
+            begin: '\\b(' + LITERALS.split(' ').join('|') + ')\\b'
+          },
+          {
+            className: 'keyword',
+            begin: '\\b(new)\\b'
+          },
+        ]
+      },
+      hljs.C_BLOCK_COMMENT_MODE,
+      STRING,
+      NUMBER,
+      {
+        className: 'literal',
```

Comment:
Why can `keywords` not be used here?

Reply:
Of all the defined keywords, only `new` is allowed in the attributes.
if there is some cleaner and easier way to make nesting, then it's beyond my knowledge.

Reply:
See above.